packages/react-components/src/localization/LocalizationProvider.tsx

```diff
@@ -174,6 +174,9 @@ export interface ComponentStrings {
   /* @conditional-compile-remove(total-participant-count) */
   /** Strings for the participant list component */
   ParticipantList: ParticipantListStrings;
+  /* @conditional-compile-remove(hide-attendee-name) */
+  /** Strings for the attendee role */
+  AttendeeRole: string;
```

Comment:
Can we make an interface AttendeeRoleStrings here? Just in case we have to add more strings for this feature

Reply:
Yeah, I agree. Will create a interface in next PR.